tools/commands.js

```diff
@@ -486,7 +486,21 @@ main.registerCommand({
     utils.validatePackageNameOrExit(
       packageName, {detailedColonExplanation: true});
 
-    var fsName = colonConverter.convert(packageName);
+    // When we create a package, avoid introducing a colon into the file system
+    // by naming the directory after the package name without the prefix.
+    var fsName = packageName;
+    if (packageName.indexOf(":") !== -1) {
+      var split = packageName.split(":");
+
+      if (split.length > 2) {
+        Console.error(packageName +
```

Comment:
Seems like this should be checked by package-version-parser instead of just here.  (It's semi checked at package publish time by troposphere... I think you can technically publish a multi-colon package if you're in mdg.)

Greenspan just added a few other rules like that at the last minute so I think that should be fine.  And doesn't affect anything existing:

```
glasser@perseid 0 ~/Projects/Meteor/meteor release-1.0.4 u= $ sqlite3 ~/.meteor/package-metadata/v2.0.1/packages.data.db
sqlite3 ~/.meteor/package-metadata/v2.0.1/packages.data.db
SQLite version 3.7.13 2012-07-17 17:46:21
Enter ".help" for instructions
Enter SQL statements terminated with a ";"
sqlite> select * from packages where name like '%:%:%';
select * from packages where name like '%:%:%';
```